console/src/state/mod.rs

```diff
@@ -386,6 +421,35 @@ impl FieldValue {
     }
 }
 
+impl Attribute {
+    fn make_formatted(
+        styles: &view::Styles,
+        attributes: &mut Vec<Attribute>,
+    ) -> Vec<Vec<Span<'static>>> {
+        let key_style = styles.fg(Color::LightBlue).add_modifier(Modifier::BOLD);
+        let delim_style = styles.fg(Color::LightBlue).add_modifier(Modifier::DIM);
+        let val_style = styles.fg(Color::Yellow);
+        let unit_style = styles.fg(Color::LightBlue);
+
+        let mut formatted = Vec::with_capacity(attributes.len());
+        let attributes = attributes.iter();
+        for attr in attributes {
+            let mut elems = vec![
+                Span::styled(attr.field.name.to_string(), key_style),
+                Span::styled("=", delim_style),
+                Span::styled(format!("{}", attr.field.value), val_style),
```

Comment:
we may want to think about only re-formatting this string when the field value _changes_, rather than every time we render a screen...